File: check_spring_server/src/main/java/com/weather/bigdata/it/app/check_spring_server/service/Judge.scala
package com.weather.bigdata.it.app.check_spring_server.service

import java.util.Date

import com.weather.bigdata.it.app.check_spring_server.utils.judge.JudgeType

trait Judge {
  def checkTime(data_time: Date, dt: Long, update_time: Date, dupt: Long): (JudgeType,String)
  def urlAliveGet(url: String): (JudgeType, String)
}

File: src/main/java/com/weather/bigdata/it/app/check_spring/service/Judge.java
package com.weather.bigdata.it.app.check_spring.service;

import com.weather.bigdata.it.app.check_spring.utils.judge.JudgeType;
import scala.Tuple2;

import java.util.Date;

public interface Judge {
    Tuple2<JudgeType,String> checkTime(Date data_time, java.lang.Long dt , Date update_time, java.lang.Long dupt );
    Tuple2<JudgeType,String> urlAliveGet(String url);
}
